test/verify/check-system-info

```diff
@@ -412,7 +412,7 @@ class TestSystemInfo(MachineCase):
         else:
             pci_selector = '#hwinfo #pci-listing'
         # PCI
-        b.wait_in_text(pci_selector + ' .listing-ct caption', "PCI")
+        b.wait_in_text(pci_selector + ' .listing-ct-heading', "PCI")
```

Comment:
This breaks [rhel-8-1-distropkg](https://209.132.184.41:8493/logs/pull-12552-20190814-061111-bbf0425d-cockpit-project-cockpit-rhel-8-0-distropkg/log.html#236)

Reply:
Just noticed. Posting a fix, thanks

Reply:
@marusak You confused me, rhel-8-1-distropkg does not exist :P It's the rhel-8-0-distropkg of course. But the point is valid. Thanks!